src/Microsoft.AspNet.Mvc.DataAnnotations/IValidationAttributeAdapterProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNet.Mvc.ModelBinding.Validation;
+using Microsoft.Extensions.Localization;
+
+namespace Microsoft.AspNet.Mvc.DataAnnotations
+{
+    /// <summary>
+    /// Provider for supplying <see cref="IAttributeAdapter"/>'s.
+    /// </summary>
+    public interface IValidationAttributeAdapterProvider
+    {
+        /// <summary>
+        /// Returns the <see cref="IAttributeAdapter"/> for the given <see cref=" ValidationAttribute"/>.
+        /// </summary>
+        /// <param name="attribute">The <see cref="ValidationAttribute"/> to create an <see cref="IAttributeAdapter"/> for.</param>
```

Comment:
Do a quick pass to ensure you aren't adding too many of these long lines (max. `120` chars).